test/kubemark/gce/util.sh

```diff
@@ -104,6 +104,21 @@ function delete-master-instance-and-resources {
   fi
 }
 
+function make-kubemark-container {
```

Comment:
I would prefer a more descriptive name, something like `build-and-push-kubemark-image` :)